--- a/plugins/decor/deco-theme.cpp	2024-10-02 09:28:48.580162250 +0200
+++ b/plugins/decor/deco-theme.cpp	2024-10-02 15:12:43.039246266 +0200
@@ -65,7 +65,9 @@
 
     auto cr = cairo_create(surface);
 
-    const float font_scale = 0.8;
+    // artist :
+    //const float font_scale = 0.8;
+    const float font_scale = 0.65;
     const float font_size  = height * font_scale;
 
     PangoFontDescription *font_desc;
